Cargo.lock

```diff
@@ -1182,6 +1182,7 @@ dependencies = [
  "glob 0.2.11 (registry+https://github.com/rust-lang/crates.io-index)",
  "habitat-eventsrv-client 0.0.0",
  "habitat-launcher-client 0.0.0",
+ "habitat-launcher-protocol 0.0.0",
```

Comment:
I think you need to re-run `cargo check` or something similar to regenerate this lock